author.tf

```diff
@@ -321,8 +321,50 @@ module "author-api" {
       {
         "name": "PUBLISHER_URL",
         "value": "${module.publisher.service_address}/publish/"
+      },
+      {
+        "name": "DYNAMO_QUESTIONNAIRE_TABLE_NAME",
+        "value": "${module.author-dynamodb.author_questionnaires_table_name}"
+      },
+      {
+        "name": "DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME",
+        "value": "${module.author-dynamodb.author_questionnaire_versions_table_name}"
       }
   EOF
+
+  task_has_iam_policy = true
+
+  task_iam_policy_json = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:Scan",
+              "dynamodb:DescribeTable",
+              "dynamodb:PutItem",
+              "dynamodb:UpdateItem",
+              "dynamodb:GetItem",
+              "dynamodb:DeleteItem"
```

Comment:
If we can delete from `questionnaires_table` should it not also delete from `questionnaire_versions`?